Flyte Scribe/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+        "name": "Ezz",
+        "id": "256009740239241216"
+    },
+    "url": "flyte.moe",
+    "description": {
+        "en": "A simple presence for edo/flyte's blog."
```

Comment:
You should probably change the Description as it should be a Description of the Website. So e.g. "edo/flyte's blog is a Blog about [...]"

Reply:
Added a more detailed description as requested in commit https://github.com/EzzPrograms/Presences/commit/f3f44d27829e503155bfc9c6e0b48a111d3327bd.